tests/layer_tests/pytorch_tests/test_search_sorted.py

```diff
@@ -0,0 +1,40 @@
+# Copyright (C) 2018-2024 Intel Corporation
+# SPDX-License-Identifier: Apache-2.0
+
+import pytest
+
+from pytorch_layer_test_class import PytorchLayerTest
+import numpy as np
+
+
+class TestSearchSorted(PytorchLayerTest):
+    def _prepare_input(self):
+        return (np.array(self.sorted).astype(self.sorted_type),np.array(self.values).astype(self.values_type))
+
+    def create_model(self, right_mode):
+        import torch
+
+        class aten_searchsorted(torch.nn.Module):
+            def __init__(self, right_mode):
+                super(aten_searchsorted, self).__init__()
+                self.right_mode = right_mode
+
+            def forward(self, sorted, values):
+                return torch.searchsorted(sorted, values, right=self.right_mode)
+
+        ref_net = None
+
+        return aten_searchsorted(right_mode), ref_net, "aten::searchsorted"
+
+    @pytest.mark.nightly
+    @pytest.mark.precommit
+    @pytest.mark.parametrize(("sorted", "values"), [([[1, 3, 5, 7, 9], [2, 4, 6, 8, 10]], [[3, 6, 9], [3, 6, 9]]),([1, 3, 5, 7, 9], [[3, 6, 9],[0, 5, 20]])])
```

Comment:
Given this op contains type promotion and input conversion done internally in torch may affect out values, maybe test data should include also few handpicked values with fractional part just to be sure?

Reply:
Good point - will prepare a malicious case just for that

Reply:
FIxed.